src/local_file_store.ts

```diff
@@ -26,6 +26,7 @@ export class LocalFileStore implements FileStore {
   }
 
   private findFile(path: string) {
+    console.log('looking for file: ', path)
```

Comment:
Probably want to change that `logger.debug` or something like that. That would "pollute" logs a bit for our users.

Reply:
I don't think it's necessary - so I just removed it. Let me know if you want me to put it back.